libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -107,6 +107,38 @@ string YulUtilFunctions::copyToMemoryFunction(bool _fromCalldata)
 	});
 }
 
+string YulUtilFunctions::copyToMemoryLiteralFunction(string const& _literal)
```

Comment:
Can you re-use this in other places?

Reply:
I think revert reason if debug has the same code like this helper. Will try using it over there.

Reply:
Hm, it can be reused in there, but I think that piece of code in revert reason if debug can't camm other functions (as it is called from inline assembly at some places). It was like that before, I'll need to re-check that.

Reply:
There's also: abiEncodingFunctionStringLiteral YulUtilFunctions::conversionFunctionSpecial and YulUtilFunctions::revertReasonIfDebug

Reply:
Yes, revertReasonIfDebug is inlined in certain places, so maybe let's keep that as it is.

Reply:
On the other hand, changing that to a function might also make sense at some point. It cannot be changed to a function everywhere, though.

Reply:
For `revertReasonIfDebug` we have #8456 and once it is done, we should use this helper. Also, this would leave link there as a reminder 
I will update this PR with usage in other functions now.

Reply:
Needed to refactor a bit, and called this function from `abiEncodingFunctionStringLiteral` and `conversionFunctionSpecial`